packages/apps-config/src/ui/logos/index.ts

```diff
@@ -34,6 +34,7 @@ import nodeRobonomics from './nodes/robonomics.svg';
 import nodeStafi from './nodes/stafi.png';
 import nodeSubsocial from './nodes/subsocial.svg';
 import nodeSubstrate from './nodes/substrate-hexagon.svg';
+import nodeEquilibrium from './nodes/equilibrium.svg';
```

Comment:
Move the import and subsequent use up to be alphabetical (makes maintenance easier)